frontend/src/metabase/public/components/EmbedFrame.jsx

```diff
@@ -33,6 +33,9 @@ type Props = {
   parameters?: Parameter[],
   parameterValues?: { [key: string]: string },
   setParameterValue: (id: string, value: string) => void,
+  setMultipleParameterValues: (parameterValues: {
+    [key: string]: string,
+  }) => void,
```

Comment:
We don't use flow types, but I guess it would be handy to have the definition here, just to make the difference from `setParameterValue` a bit more explicit